caffe2/contrib/fakelowp/layernorm_fp16_fake_op.h

```diff
@@ -75,7 +79,49 @@ class LayerNormFakeFp16Op final : public Operator<Context> {
         mean_data,
         sigma_data,
         &context_);
+//    ComputeSigmaAndFusedParams<T>(
+//       M, epsilon_, mean_data, sigma_data, sigma_data, scale_data, bias_data);
+    ConstEigenVectorArrayMap<T> var_arr(sigma_data, M);
+    EigenVectorArrayMap<T> sigma_arr(sigma_data, M);
+    sigma_arr = var_arr + static_cast<T>(epsilon_);
+    math::Rsqrt<T, CPUContext>(M, sigma_data, scale_data, &context_);
+    math::Mul<T, CPUContext>(M, scale_data, sigma_data, sigma_data, &context_);
+    EigenVectorArrayMap<T>(bias_data, M) =
+        -ConstEigenVectorArrayMap<T>(scale_data, M) *
+        ConstEigenVectorArrayMap<T>(mean_data, M);
 
+//    LayerNormForward<T>(
```

Comment:
were you planning to abstract this into this function? also with ComputeSigmaAndFusedParams? i think it is a good idea to do so

Reply:
I will abstract this into a function.